app/views/users/ticket_sales.py

```diff
@@ -81,23 +81,33 @@ def display_ticket_stats(event_id):
     for order in orders:
         if order.status == 'initialized':
             order.status = 'pending'
+        fees = DataGetter.get_fee_settings_by_currency(DataGetter.get_event(order.event_id).payment_currency)
         orders_summary[str(order.status)]['orders_count'] += 1
         orders_summary[str(order.status)]['total_sales'] += order.amount
+
         for order_ticket in order.tickets:
             discount = TicketingManager.get_discount_code(event_id, order.discount_code_id)
             orders_summary[str(order.status)]['tickets_count'] += order_ticket.quantity
             ticket = get_ticket(order_ticket.ticket_id)
             tickets_summary[str(ticket.id)][str(order.status)]['tickets_count'] += order_ticket.quantity
+            ticket_price = ticket.price
+            if fees and not ticket.absorb_fees:
+                order_fee = fees.service_fee * (ticket.price * order_ticket.quantity) / 100
```

Comment:
please make 100.0 just to ensure that it is always float value that is returned.